state-chain/runtime/src/lib.rs

```diff
@@ -272,19 +278,30 @@ impl pallet_cf_vaults::Config<BitcoinInstance> for Runtime {
 	type CfeMultisigRequest = CfeInterface;
 }
 
+impl pallet_cf_vaults::Config<Instance4> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Chain = Arbitrum;
+	type SetAggKeyWithAggKey = cf_chains::arb::api::ArbitrumApi<EvmEnvironment>;
+	type Broadcaster = ArbitrumBroadcaster;
+	type WeightInfo = pallet_cf_vaults::weights::PalletWeight<Runtime>;
+	type ChainTracking = ArbitrumChainTracking;
+	type SafeMode = RuntimeSafeMode;
+	type CfeMultisigRequest = CfeInterface;
+}
+
 use chainflip::address_derivation::AddressDerivation;
 
-impl pallet_cf_ingress_egress::Config<EthereumInstance> for Runtime {
+impl pallet_cf_ingress_egress::Config<Instance1> for Runtime {
```

Comment:
? this seems like a regression? We were previously able to use nice names

Reply:
I had to move the ```PalletInstanceAlias``` from runtime crate to chains crate because of trait bounds in CFE. I dont know why type aliases dont work after that. Therefore, I had to resort to uses raw types. Maybe I was missing something so feel free to try it out.